lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+  attr_accessor :patrons
+  attr_reader :name, :capacity
+  def initialize(name, int)
+    @name = name
+    @capacity = int
+    @patrons = []
+  end
+
+  def add_patron(i)
+    self.patrons << i.to_s
+  end
+
+  def yell_at_patrons
+    modified_array = []
+
+    self.patrons.each do |i|
+      modified_array << i.upcase
+    end
+
+    return modified_array
+  end
+
+  def over_capacity?
+    if self.patrons.size <= self.capacity
```

Comment:
Nice job using the `if/else` here. When is our venue over_capacity? Would it be when our patrons array is <= or when it is >? 

You could also write this without `if/else` and you don't need the `self`